interface/vsomeip/application.hpp

```diff
@@ -18,6 +18,13 @@
 #include <vsomeip/constants.hpp>
 #include <vsomeip/handler.hpp>
 
+namespace boost {
+namespace asio {
```

Comment:
How is this intended to work for "old" boost versions that define io_service, but do not define io_context?

Reply:
these are just forward declarations. They will be linked against boost in the library itself, irrespectively what boost on host application or it might not be used in host at all (the latest point  was checked)